manifests/init.pp

```diff
@@ -81,12 +80,12 @@
 #
 # @param arch
 #   cpu architecture
+# @param bin_dir
+#   location of the nomad binary
```

Comment:
I'd keep the old parameter instead of moving it around.

Reply:
done